.github/workflows/rust.yml

```diff
@@ -18,5 +18,9 @@ jobs:
       - uses: actions/checkout@v2
       - name: Build
         run: cargo build --verbose
+      - name: Build without default features
```

Comment:
I think it's probably better to order the steps such we have: build and test with default features and then build and test without default features (test phase should find more stuff already compiled by the build phase)

Reply:
Ok, Initially, I thought about adding a feature matrix like https://github.com/bitcoindevkit/bdk-cli/blob/master/.github/workflows/cont_integration.yml  But that didn't seem to work well with "--no-default-features" and I didn't want to complicate stuff.